examples/protobuf/README.md

```diff
@@ -0,0 +1,25 @@
+# Simple Hello World Pub/Sub implementation
+ In order for this demo to work, you must have your AWS credentials present in the form that the [AWS SDK for Go](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/sessions.html) requires.
+
+1. Create the person.pb.go files for both publisher and subscriber:
+```
+cd ~/go/src/github.com/infobloxopen/atlas-pubsub/examples/protobuf
```

Comment:
It would be better if you start the path from atlas-pubsub as not everyone has all their code under the go/src/github.com/infoblox path.

Reply:
okay, I was just following the template of the readme in the [hello example](https://github.com/infobloxopen/atlas-pubsub/tree/master/examples/hello). How would you like me to format it - something like the following?

`cd <your-path-here>/atlas-pubsub/examples/protobuf`

Reply:
$GOPATH may be an appropriate stand-in

Reply:
for me, that would be `$GOPATH/src/github.com/infoblox/atlas-pubsub/examples/protobuf` - is that generic enough?